SignalServiceKit/src/Storage/TSDatabaseView.m

```diff
@@ -144,7 +144,7 @@ + (void)asyncRegisterUnseenDatabaseView
 
     [self registerMessageDatabaseViewWithName:TSUnseenDatabaseViewExtensionName
                                  viewGrouping:viewGrouping
-                                      version:@"1"
+                                      version:@"2"
```

Comment:
I agree it's good to keep the read status updated as you have. I was just thinking for paranoia incase we add a new dynamic interaction and forget that we have to have a line in each initializer to mark it as read - but no biggie.